blog/2023-08-23-nfdi-grant.md

```diff
@@ -0,0 +1,20 @@
+---
+author: Martin Magdinier
+title: "OpenRefine receives a new NFDI grant to improve its reconciliation features"
```

Comment:
Given the large font size used in our blog I think it's better to use shorter titles, such as:
```suggestion
title: "NFDI funds reconciliation improvements"
```